packages/shared/openapi/internalapi/openapi.yaml

```
@@ -11,6 +11,8 @@ paths:
     $ref: './paths/users.yaml#/users_{user_id}'
   /internal/v1/plats/{plat_id}:
     $ref: './paths/plats.yaml#/plat_{plat_id}'
+  /internal/v1/plats/{plat_id}/like:
```

favoriteに統一しましょうか
1つのplatには複数のfavoriteが紐づくので、ここでは以下のようになりますね

```suggestion
  /internal/v1/plats/{plat_id}/favorites:
```

---

修正しました。